tools/default.nix

```diff
@@ -0,0 +1,75 @@
+{
+  pkgs ? import <nixpkgs> {},
+  lib ? pkgs.lib,
+
+  content ? null, # path to content repository
+  withDeveloperMode ? true,
```

Comment:
I don't know nix well, but as that looks like user-configurable builds, I'd recommend adding support for `USE_SDL_MAINLOOP` as well if someone wants to run the game natively under Wayland, as `WITH_DEVELOPER_MODE` forces the game to run under X11 so that the developer mode works.

Reply:
I added support for that now. `USE_SDL_MAINLOOP` can now be activated with `nix-build tools/default.nix --arg useSDLMainloop true --arg withDeveloperMode false` and it removes the X11 libs appropriately.